graze/pistachio
# Improve footer versatility and implementation.

**Summary**

- removed the need for as many helper classes
- added the ability to support right aligned content (i.e. a copyright notice) in the secondary footer row

**Notable Changes**

- footer sections no longer need helper classes `margin-reset` or `list-inline`
- secondary footer links are centred on mobile
- the secondary footer section is set to display block, to achieve 100% width

**Upgrade Steps**

- this should be non-breaking as the helper classes could be left behind and it should work the same, I believe.

**QA**

- Check the footer including the example markup with copyright notice on various devices.
- Confirm this is a non-breaking change if the helper classes are left on, and there are no other required markup changes.

**Screenshots**

<img width="368" alt="screen shot 2016-11-18 at 12 00 48" src="https://cloud.githubusercontent.com/assets/10405691/20429690/24cb6794-ad88-11e6-9f17-cf9d80a5dc32.png">

![screen shot 2016-11-18 at 12 00 54](https://cloud.githubusercontent.com/assets/10405691/20429702/2fe28ae0-ad88-11e6-92b7-d4052c440c1e.png)
